### ISSUE TITLE
KubeCon+CloudNativeCon EU 2024 - OTel Integrations Panel Discussion

### ISSUE THREAD

**Issue by svrnm** [open-telemetry/community#1810]
Via https://github.com/open-telemetry/community/issues/1795#issuecomment-1815959874 & https://github.com/open-telemetry/community/issues/1795#issuecomment-1817319090

> Observability Included: Integrating OTel into your project: We see a raise of oss projects [integrating otel](https://opentelemetry.io/ecosystem/integrations/), so we should take the opportunity to talk about them, but also about best practices how to do it for projects which do not yet have o11y built-in

> I feel like the second might be a good panel @svrnm. Not sure how kubecon talks work for that but if we got 3-4 services to chat about it and then I or someone else could moderate

**Comment by svrnm** [comment]
> I can go ahead and ping some people if necessary. I’m guessing we’d need speakers lined up for the proposal? Initial thought is to start with nginx, tyk, and VMware

@cartersocha we also have 2 teams at Cisco (Thousand Eyes & NSO) that have done an integration. I can check with folks there as well if you like. Otherwise, I wonder if we should also see if we can get some contributors from CNCF projects onboard that have done their OTel integration

**Comment by cartersocha** [comment]
Yeah that’d be great! I know @alexanderwert mentioned elastic added support in their SDKs which might be relevant too. 

Reviewing our integration list I don’t personally know anyone at the other cncf projects. Do you have an idea where to start? Kubernetes would be nice given the event title haha

**Comment by cartersocha** [comment]
I see @dashpole made the most recent commit to their docs. David do you know any of the kubernetes contributors who worked on this / was this your implementation?

**Comment by dashpole** [comment]
This was my implementation.  I wasn't planning on going to kubecon, EU, though.

**Comment by AlexanderWert** [comment]
> Yeah that’d be great! I know @AlexanderWert mentioned elastic added support in their SDKs which might be relevant too.

We planned to submit a talk on that topic anyways, sharing some learnings we had from our baked-in instrumentation in the Elasticsearch clients for different languages. We would be happy to participate in the panel discussion (cc @estolfo).

**Comment by svrnm** [comment]
Pinging the authors of the integrations blog posts we had recently: @davidgs @ctlong @sonjachevre -- would anyone of you be interested in participating?

**Comment by SonjaChevre** [comment]
I would be happy to join!

**Comment by svrnm** [comment]
@cartersocha do you want to moderate this panel, or shall I?

**Comment by davidgs** [comment]
I would suggest @orishoshan, the Otterize CTO for this, but I would be happy to do it as a last-resort.

**Comment by svrnm** [comment]
> I would suggest @orishoshan, the Otterize CTO for this, but I would be happy to do it as a last-resort.

whoever can speak best about that otel integration 👍

**Comment by svrnm** [comment]
Ok, looks like we will have the panel of integrators with Tyk, Otterize, Elastic and NSO & me or @cartersocha as moderator. Let's try to get this list finalised by Friday such that we can insert it into the CfP for KubeCon/Observability Day

**Comment by orishoshan** [comment]
Sure, happy to! Would love to help more people get started with OTel, and this would help get the word out there :)

**Comment by mkocher** [comment]
@ctlong is on a plane today so responding for him. Either one of us would love to be on the panel and represent VMware’s TAS integration. We’ve been really impressed with the ease of integrating OTel and how welcoming the community has been. Neither of us were planning on attending so will need to ask for funding before we can confirm.

**Comment by cartersocha** [comment]
> @cartersocha do you want to moderate this panel, or shall I?

I’d love to moderate! I can submit the CFP too. Is there a specific submission portal for observability day? This would def fit under both

**Comment by svrnm** [comment]
> > @cartersocha do you want to moderate this panel, or shall I?
> 
> I’d love to moderate! I can submit the CFP too. Is there a specific submission portal for observability day? This would def fit under both

Here is the CfP: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/cfp-colocated-events/ 

The CfP for Observability Day closes a little bit later than the one for KubeCon: _CFP Close: Sunday, 3 December at 11:59pm Central European Time (UTC+1)_, 

But it would be good to get this done quickly.

**Comment by plajjan** [comment]
I could participate on the panel sharing our experiences with integrating Cisco NSO observability using OpenTelemetry. Note that it is conditioned on manager approval for travel etc.

**Comment by danielgblanco** [comment]
Not sure if I can be of much help here because I'm more on the adoption side, but would love to see this panel talk about best practices for instrumentation: what to instrument and what not, what signal and level of granularity, how OTel ultimately leaves config on certain things up to the end-user with span processors or metric views, etc.

**Comment by cartersocha** [comment]
@plajjan , @mkocher / @ctlong

We will need speaker info from yall if you can share that over slack. I think if we get your email we can invite you and then you fill it out yourself. Is that correct @svrnm ?

**Comment by estolfo** [comment]
Hi all, are we intending this panel discussion to be for Observability Day or Kubecon mainstream?

**Comment by estolfo** [comment]
Yes, that's why I was asking. Observability Day might be a better context for this topic anyway.

**Comment by svrnm** [comment]
> @plajjan , @mkocher / @ctlong
> 
> We will need speaker info from yall if you can share that over slack. I think if we get your email we can invite you and then you fill it out yourself. Is that correct @svrnm ?

Yes that is correct! So **if** you want to speak, please provide your email address to @cartersocha or me (via slack or any other way you prefer if you don't want to share it here on this issue)

So far we have @SonjaChevre and @estolfo confirmed

**Comment by cartersocha** [comment]
What is the cfp due date for observability day @austinlparker

**Comment by svrnm** [comment]
For those of you who have not shared their email address with me or @cartersocha please do so by end of today PST (Thursday), such that we can send the proposal tomorrow

**Comment by cartersocha** [comment]
We’re just missing @plajjan ’s now

**Comment by cartersocha** [comment]
I pinged the others

**Comment by cartersocha** [comment]
Submitted the talk. @orishoshan, @mkocher, @estolfo, @SonjaChevre yall should get an email
